alert-database/src/test/java/com/synopsys/integration/alert/database/api/mock/MockSystemStatusRepository.java

```diff
@@ -0,0 +1,141 @@
+package com.synopsys.integration.alert.database.api.mock;
+
+import java.util.Date;
+import java.util.List;
+import java.util.Optional;
+
+import org.springframework.data.domain.Example;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+
+import com.synopsys.integration.alert.database.system.SystemStatus;
+import com.synopsys.integration.alert.database.system.SystemStatusRepository;
+
+public class MockSystemStatusRepository implements SystemStatusRepository {
+
+    //Only methods that are used by a test are currently implemented, all others are left default.
+    SystemStatus systemStatus = new SystemStatus(Boolean.FALSE, new Date());
```

Comment:
Same as my comment above. We should make this field private and expose a setter for this method. We should also avoid hardcoding the fields. This reduces the usability of the mock object in the tests.